$NetBSD: patch-af,v 1.5 2004/03/30 15:13:12 tv Exp $

Parts from:
ftp://ftp.kame.net/pub/kame/misc/gnupg-1.2.2-IPv6.diff.gz

--- configure.orig	Tue Dec 23 14:28:22 2003
+++ configure
@@ -4601,7 +4601,7 @@ extern "C" {
 
 EOF
         # Now generate the symbol file.
-        sed 's/^.* _\?\(.*\)$/extern char \1;/' < "$ac_nlist" >> conftest.c
+        sed 's/^.* _\{0,1\}\(.*\)$/extern char \1;/' < "$ac_nlist" >> conftest.c
 
         cat <<EOF >> conftest.c
 #if defined (__STDC__) && __STDC__
@@ -4621,7 +4621,7 @@ struct {
 dld_preloaded_symbols[] =
 {
 EOF
-        sed 's/^_\?\(.*\) _\?\(.*\)$/  {"\1", (__ptr_t) \&\2},/' < "$ac_nlist" >> conftest.c
+        sed 's/^_\{0,1\}/\(.*\) _\{0,1\}\(.*\)$/  {"\1", (__ptr_t) \&\2},/' < "$ac_nlist" >> conftest.c
         cat <<\EOF >> conftest.c
   {0, (__ptr_t) 0}
 };
@@ -13579,7 +13579,7 @@ done
 
 
 
-for ac_func in waitpid wait4 sigaction sigprocmask rand pipe stat
+for ac_func in waitpid wait4 sigaction sigprocmask rand pipe stat getaddrinfo
 do
 as_ac_var=`echo "ac_cv_func_$ac_func" | $as_tr_sh`
 echo "$as_me:$LINENO: checking for $ac_func" >&5
